CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,69 @@
+# Code of Conduct
+
+## Our Pledge
+
+We, as members and maintainers of the **TwinTrim** project, pledge to foster a welcoming, inclusive, and harassment-free environment for everyone involved. Whether you are contributing code, writing documentation, submitting issues, or participating in discussions, we are committed to making the TwinTrim community a positive experience for individuals of all ages, body sizes, visible or invisible disabilities, cultural backgrounds, ethnicities, gender identities and expressions, levels of experience, education, socioeconomic status, nationalities, personal appearances, races, religions, or sexual identities and orientations.
```

Comment:
Again, there is only one maintainer here, and this is extremely lengthy and unnecessary.